site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -500,10 +569,8 @@ pages.
 ### Notifications
```

Comment:
Can you also fix the highlighted code? It should highlight the "notifications" permissions